multicast/recv.py

```
@@ -211,6 +212,85 @@
+
+_mcast_recv_lazy_bind_warning_message = "\n".join([
+	"Lazy call to multicast.joinstep with unspecified bind group.",
+	f"Will bind to {multicast._MCAST_DEFAULT_BIND_IP}.",  # skipcq: PYL-W0212 - module ok
+	"Tip: Pass a value for bind_group to suppress this message",
+	"(such as 'bind_group=multicast._MCAST_DEFAULT_BIND_IP')",
+])
+
+_mcast_recv_join_only_multicast_warn_msg = "\n".join([
+	_mcast_recv_bind_not_join_prefix,
+	"Just use socket.Socket.bind(...) for non-multicast networking.",
+])
```

Ruff (0.8.2)

Consider f-string instead of string join

Replace with f-string

(FLY002)